Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM ubuntu:bionic
 
+ARG USER_ID=1000
```

Comment:
please move this back where it was, immediately before its use. moving it to the top reduces caching in the dockerfile.

Reply:
I don't think this will reduce any cacheing, actually may improve it since these lines are more stable than the next lines (installing a bunch of stuff).

I mostly just like having all the arguments at the top for scanability. I can move it back if you really want, but don't see the need.

Reply:
Everything after the ARG line depends on the value of the argument, so if the user id changes, we will have to reinstall all the debian packages and stack.

Reply:
Ok, I moved it back.